src/source/source_packet_ros.hpp

```diff
@@ -201,7 +201,7 @@ inline void DestinationPacketRos::init(const YAML::Node& config)
       ros_send_topic, "rslidar_packets");
 
   nh_ = std::unique_ptr<ros::NodeHandle>(new ros::NodeHandle());
-  pkt_pub_ = nh_->advertise<rslidar_msg::RslidarPacket>(ros_send_topic, 10);
+  pkt_pub_ = nh_->advertise<rslidar_msg::RslidarPacket>(ros_send_topic, 100);
```

Comment:
This would be the only real change:

Introduced in: https://github.com/RoboSense-LiDAR/rslidar_sdk/commit/d9d6b9b453e28f15db0bab3f4746440064d91ea3
> feat: increase send and receive buffer to avoid packet loss